src/vs/workbench/contrib/search/common/searchModel.ts

```diff
@@ -560,10 +564,9 @@ export class FolderMatch extends Disposable {
 		if (fileMatch) {
 			fileMatch.bindModel(model);
 		} else {
-			const folderMatches = this.folderMatchesIterator();
-			for (const elem of folderMatches) {
-				elem.bindModel(model);
-			}
+			const folderMatch = this.getFolderMatch(model.uri);
+			const match = folderMatch?.hasFileUriDownstream(model.uri);
```

Comment:
I guess I just noticed this but I'd expect a `has` method to return a boolean. If it returns a match, I'd expect it to start with `get` or something similar